data/mods/Magiclysm/enchanted/enchanted_tools.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "id": "copper_bracelet_manafocus_off",
+    "type": "ARMOR",
+    "name": "copper mana focusing bracelet",
+    "description": "An ornate copper bracelet covered in magical symbols, crafted and manipulated to act as a focus for magical powers when you are enchanting. This is the second most basic enchanting tool. Activate to say the command word and turn the bracelet into a pencil like tool with a sharp point that you can use to etch magical symbols into other materials.",
+    "weight": 30,
+    "volume": 0,
+    "price": 25000,
+    "material": [ "copper" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "flags": [ "FANCY" ],
+    "use_action": [
+      {
+        "type": "transform",
+        "target": "copper_bracelet_manafocus_on",
+        "active": false,
+        "msg": "You say the command and your bracelet straightens out into a tool.",
+        "menu_text": "Switch tool mode",
+        "moves": 50
+      }
+    ]
+  },
+  {
+    "id": "copper_bracelet_manafocus_on",
+    "type": "TOOL",
+    "name": "copper mana focuser",
+    "description": "An ornate copper rod with a fine sharp point covered in magical symbols, roughly the size of a pencil.  Crafted and manipulated to act as a focus for magical powers when you are enchanting.  This is the second most basic enchanting tool, think of it as a magical screwdriver.  You use the sharp point to etch symbols and runes into the substance of what you are enchanting.",
+    "weight": 30,
+    "volume": 0,
+    "price": 25000,
+    "cutting": 4,
+    "to_hit": 1,
+    "material": [ "copper" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "qualities": [ [ "MANA_FOCUS", 1 ], [ "CUT_FINE", 1 ] ],
```

Comment:
this is a repeat of an item already in game, maybe bump this to mana focus 2, since it has the transform ability and everything? otherwise it's strictly worse than the other one, which is wearable.

Reply:
It's not a repeat, he had this exact item, both of them in fact, in a zip I gave him, with the same descriptive text explaining the item's use and purpose, so I don't know why he made an entirely new one. 
This is wearable, you just can't use it as a tool in the wearable form because that doesn't make sense. All it really does is straighten itself out from a bent form into a short sharpened wand rather than a circular bracelet so that you can hold it and use it. I made it that way to be neat-seeming to the players and also not be a hassle to carry around, I've never seen a tool you can use with your wrist or forehead, we have hands for tools, and the tools should probably be touching the object. 
I would suggest doing it the other way around, as a circlet or whatever headgear would be bigger than a bracelet, more materials, more ornate, etc.. that should be rank 2.